ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCliBase.scala

```diff
@@ -213,6 +213,15 @@ class CommonCliBase(name: LedgerName) {
         )
         .action((eventsPageSize, config) => config.copy(eventsPageSize = eventsPageSize))
 
+      opt[Int]("events-decoding-parallelism")
```

Comment:
```suggestion
      opt[Int]("buffers-prefetching-parallelism")
```
Let's be more specific, since not all streaming queries use parallel fetch/decode